File: mo-member/mo-member-api/src/main/java/ink/ckx/mo/member/api/model/query/UserPageQuery.java
package ink.ckx.mo.member.api.model.query;

import ink.ckx.mo.common.core.base.BasePageQuery;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/18
 */
@Schema(description = "会员用户分页Query")
@Data
public class UserPageQuery extends BasePageQuery {

    @Schema(description = "关键字")
    private String keywords;
}

File: mo-member/mo-member-api/src/main/kotlin/ink/ckx/mo/member/api/model/query/UserPageQuery.kt
package ink.ckx.mo.member.api.model.query

import ink.ckx.mo.common.core.base.BasePageQuery
import io.swagger.v3.oas.annotations.media.Schema

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/18
 */
@Schema(description = "会员用户分页Query")
data class UserPageQuery(

    @Schema(description = "关键字")
    var keywords: String? = null
) : BasePageQuery()